### PR TITLE
Fixed bug in Product Export form which resulted in wrong input fields being shown when changing time condition

### PR DESCRIPTION
**Pull request by Schwierig**
| Q | A |
| --- | --- |
| Added Specs |  |
| Added Behats |  |
| Changelog updated | (no) |
| Review and 2 GTM |  |
| Micro Demo to the PO (Story only) |  |
| Migration script |  |
|  Tech Doc |  |

Wasn't really sure what to do with the changelog, would be glad to change the request after further instruction ;)

### CODE REVIEW

**Comment by fabienlem** [comment]
Hello @Schwierig 

Thank you a lot for your contribution and your reactivity :)

The fix is OK for me, for avoiding regressions in the future, we test every fix with at least a behat scenario. Do you feel like writing it? If not, I will do it for you. But in this case you have to send your pull request directly in our repository.

You can add this scenario to the following feature file:
features/export/product-export-builder/export_products_by_specific_date.feature

You may use the "Update the updated time condition field" scenario as an example.

I will also fill the changelog file for you. 

Regards

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/filter/product/updated.js [hunk 1]
```diff
@@ -124,9 +124,9 @@ define([
                 value = '';
             }
 
-            if ('SINCE LAST JOB' !== operator && 'SINCE LAST N DAYS' !== operator) {
+            if ('<' === operator) {
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/filter/product/updated.js]: Are you sure it shouldn't be `'>'` here?
See https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/filters.yml#L31

**Comment by fabienlem** [comment]
As @phaseinducer said, I made a typo in the hint I gave you. The operator to use is '>'.
Sorry about that!

**Comment by Schwierig** [comment]
Ah okay, seems right. Will change the wrong operator and will try to write the behat scenario myself.

**Comment by fabienlem** [comment]
Hello @Schwierig,

Is everything fine? Don't hesitate to ping me if you need some help :)

Regards

**Comment by Schwierig** [comment]
Yeah, everything is fine. We just have some pressing issues at work that I need to deal with.

**Comment by Schwierig** [comment]
Got some free time today, so I fixed the change request. But I'm not sure what Behat tests I should here, since the functionality tested in the "export_products_by_specific_date.feature"-file reflects the use case how it should be and how it works now.

**Comment by Schwierig** [comment]
closed by accident...

**Comment by nidup** [comment]
Hi @Schwierig 

Thank you for the fix, yes this case seems already well covered by behat.

I think you had an issue with your "git rebase" as a lot of commits (not only yours) appears in this PR, may you check that you properly fetched the master before to rebase your branch?

BTW, to be able to merge your PR I need that you read and sign our contributor agreement https://www.akeneo.com/contributor-license-agreement/ 

Thank you!

**Comment by Schwierig** [comment]
Sorry, didn't do much rebasing in the last months. Also sorry it took so long for such a small fix.
Should be fixed now.

**Comment by fabienlem** [comment]
Hello @Schwiering,

Don't worry about it, we appreciate your help! The fix is good to me. We have to run some continuous integration checks and we will be able to merge the PR. I will keep you updated. As nidup said, don't forget to sign our contributor agreement :)

Regards

**Comment by fabienlem** [comment]
Hello @Schwierig,

I took some time, but I'm glad to announce you that your contribution is merged! (I cherry picked your commit in this other PR #5198 in order to modify the change log and to rebase it on the 1.6 branch). Your fix will be available in the next PIM release!

Thank you again for your contribution :+1: 

Best regards!
Fabien

**Pull request closed without merging**